tests/test_nodes.py

```diff
@@ -660,6 +660,64 @@ def test_decay_queue(self):
         }
         self.assertDictAlmostEqual(d4, tank.storage, 15)
 
+    def test_overrides(self):
+        # node - no need to test
+        # tank
+        tank = Tank(capacity=10, area=8, datum = 4)
+        tank.apply_overrides({'capacity': 3,
+                        'area': 2,
+                        'datum': 3.5})
+        self.assertEqual(tank.capacity, 3)
+        self.assertEqual(tank.area, 2)
+        self.assertEqual(tank.datum, 3.5)
```

Comment:
Maybe something like this just to ensure the behaviour is working as intended:

```suggestion
        self.assertEqual(tank.datum, 3.5)
        self.assertEqual(tank.get_excess()['volume'], 3)
```